CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 2.8.12)
 project(libyang C)
 
 include(GNUInstallDirs)
+INCLUDE(CheckSymbolExists)
```

Comment:
Just a detail, please keep `INCLUDE` lowercase so that we do not mix it.

Reply:
Indeed, will update.